pages/[contract]/[tokenId]/index.tsx

```diff
@@ -206,12 +206,16 @@ export const getStaticProps: GetStaticProps<{
 }> = async ({ params }) => {
   const contract = params?.contract?.toString()
   const tokenId = params?.tokenId?.toString()
+  let collectionAddress = COLLECTION
+  if (COLLECTION && COLLECTION.includes(':')) {
```

Comment:
don't think you need to check for : in this case. can just split it and it will be the whole thing if there isn't a colon

Reply:
You're right, forgot about that. Ok, changed it